go/vt/wrangler/keyspace.go

```diff
@@ -917,7 +919,7 @@ func (wr *Wrangler) updateFrozenFlag(ctx context.Context, shards []*topo.ShardIn
 // the tablet was actually drained. At later times, a QPS rate > 0.0 could still
 // be observed.
 func (wr *Wrangler) WaitForDrain(ctx context.Context, cells []string, keyspace, shard string, servedType topodatapb.TabletType,
-	retryDelay, healthCheckTopologyRefresh, healthcheckRetryDelay, healthCheckTimeout, initialWait time.Duration) error {
```

Comment:
hmm, why have we removed this parameter?

Reply:
#10278 deletes WaitForDrain, so it doesn't really matter what we do here any more.